ecs-agent/acs/session/payload_responder.go

```diff
@@ -0,0 +1,162 @@
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//	http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package session
+
+import (
+	"fmt"
+
+	"github.com/aws/amazon-ecs-agent/ecs-agent/acs/model/ecsacs"
+	"github.com/aws/amazon-ecs-agent/ecs-agent/logger"
+	"github.com/aws/amazon-ecs-agent/ecs-agent/logger/field"
+	"github.com/aws/amazon-ecs-agent/ecs-agent/wsclient"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/pkg/errors"
+)
+
+const (
+	PayloadMessageName = "PayloadMessage"
```

Comment:
It seems this variable is only used in log messages. I supposes this is to avoid duplication when we have another task message we are going to handle, but those logging all reside in payloadResponder's methods, this does not provides potential to extend, which seems unnecessary to have this constant and format the string where it can be just a string. e.g. line 60.

Reply:
Currently there aren't any plans to handle other types of ACS messages in the payload responder (see this comment: https://github.com/aws/amazon-ecs-agent/pull/3840#discussion_r1288885624).

However, my thought is that in the event that ACS/we ever want to change the name of the [existing `PayloadMessage` struct](https://github.com/aws/amazon-ecs-agent/blob/dev/ecs-agent/acs/model/ecsacs/api.go#L1376), we would only need to make that change on L28 of this file using this const. But if we get rid of this const and just type the raw string "PayloadMessage" inline wherever we need it, then currently we would need to make changes in many places such as L60, L64, L73, L97, L104, L119, L125 (and the number of lines could grow to be more in the future if we need to add more log statements that require the message name).

Reply:
I do not have a strong opinion for this one. You can do whichever you prefer.